config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "1.5.0",
+    "version": "1.5.2",
```

Comment:
Is there any reason why it should be 1.5.2 instead of 1.5.1? The latest stable version released is 1.5.0.

Reply:
The version was wrong in this file.  The project has already been tagged for 1.5.1